src/orquestra/sdk/_base/_workflow.py

```diff
@@ -658,6 +659,14 @@ def wf():
 
     will raise a NotATaskWarning on the `np.ones` call, but not for the `my_task` call.
     """
+    workflow_default_dependency_imports: Optional[Tuple[Import, ...]]
+
+    if default_dependency_imports is None:
+        workflow_default_dependency_imports = None
+    elif isinstance(default_dependency_imports, ImportTypes):
+        workflow_default_dependency_imports = (default_dependency_imports,)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Can you add a test case for this?